### PR TITLE
Bumps version to 1.2.0

### PR DESCRIPTION
**Pull request by cgpu**
- Adds better handling for SSL verification
- Adds request_interval parameter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ cloudos job run --help
 ```console
 CloudOS python package: a package for interacting with CloudOS.
 
-Version: 1.0.0
+Version: 1.2.0
 
 CloudOS job functionality: run and check jobs in CloudOS.
 
-Usage: cloudos job run [OPTIONS]
+Usage: python -m cloudos job run [OPTIONS]
```
Comment (cgpu) [inline README.md:84]:
```Suggestion
Usage: python -m cloudos job run [OPTIONS]

> NOTE: You may need to include the folder of the local clone of the `cloudos-cli` repo folder in `PATH` to be able to use the command `cloudos` without calling python, for example: `export PATH="~/path-of-cloudos-cli/:$PATH"`. This is required only if the installation location is not already included in `PATH`.
```

Comment (cgpu) [inline README.md:84]:
```Suggestion
Usage: cloudos job run [OPTIONS]
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # cloudos
 
 __Date:__ 2022-07-28\
```
Comment (dapineyro) [inline README.md:3]:
```suggestion
__Date:__ 2022-10-28\
```

File: README.md [hunk 3]
```diff
@@ -41,6 +41,7 @@ pip install -r requirements.txt
 pip install .
 ```
 
+> NOTE: To be able to call the `cloudos` executable, ensure that the local clone of the `cloudos-cli` folder is included in the `PATH` variable ,using for example the command `export PATH="/aboslut/path/to/cloudos-cli:$PATH"`.
```
Comment (dapineyro) [inline README.md:44]:
```suggestion
> NOTE: To be able to call the `cloudos` executable, ensure that the local clone of the `cloudos-cli` folder is included in the `PATH` variable ,using for example the command `export PATH="/absolute/path/to/cloudos-cli:$PATH"`.
```

**Review by dapineyro — CHANGES_REQUESTED** [review]
Looks good, only two minor changes in README.md

**Comment by cgpu** [comment]
> Looks good, only two minor changes in README.md

thank you @dapineyro, addressed both, feel free to squash on your end!

**Review by dapineyro — APPROVED** [review]
LGTM! :+1:

**Pull request merged**